### PR TITLE
Cosmetic changes: added types

### PR DESCRIPTION
**Pull request by Ghostvv**
**Proposed changes**:
- added types

**Status (please check what you already did)**:
- [x] made PR ready for code review
- [ ] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog

### CODE REVIEW

File: rasa_nlu/classifiers/embedding_intent_classifier.py [hunk 1]
```diff
@@ -27,6 +22,10 @@
 
 if typing.TYPE_CHECKING:
     import tensorflow as tf
+    from rasa_nlu.config import RasaNLUModelConfig
```
Comment (akelad) [inline rasa_nlu/classifiers/embedding_intent_classifier.py]: usually all that stuff gets imported where it was before. not sure which is the correct way to do it though :P

Comment (Ghostvv) [reply rasa_nlu/classifiers/embedding_intent_classifier.py]: I noticed in rasa_core files all rasa_core imports for types were imported in such if. I don't know what is the right way as well.

Comment (akelad) [reply rasa_nlu/classifiers/embedding_intent_classifier.py]: interesting. @amn41 thoughts?

Comment (Ghostvv) [reply rasa_nlu/classifiers/embedding_intent_classifier.py]: not in all rasa_core files though

Comment (amn41) [reply rasa_nlu/classifiers/embedding_intent_classifier.py]: this is required if you are using types from optional dependencies

**Pull request merged**